components/Organisms/MyPage/Archive/Detail/MyArchiveDetailCardHeader.tsx

```diff
@@ -31,15 +34,8 @@ export default function MyArchiveDetailHeaderInfo(
   };
 
   const archiveLink = () => {
-    return router.push({
-      pathname: `/archive/update`,
-      query: {
-        id: id,
-        title: props?.title,
-        thumbnailImageUrl: '',
-        checked: true,
-      },
-    });
+    const decode = decodeURIComponent(`/archive/update?id=${archiveId}`);
+    return router.push(`${decode}&exhibitionId=${itemId}`);
```

Comment:
```suggestion
    return router.push({
      pathname: `/archive/update`,
      query: {
        id: id
      },
    });
```

로 주셔도 될 듯합니다.

제가  상세페이지>아카이브 수정하기 버튼에서 decodeURIComponent가 필요했던 이유는 api에서 link를 주고 해당 link로 넘어가기 위함이었습니다. 그러나 정아님께서는 프론트에서 `/archive/update`로 이동하고 있으니 `pathname`과 `query`로도 충분할 듯 합니다.

Reply:
아하 저는 router를 쓸때 무조건 디코딩을 해야하는 줄 알았어요 알려주셔서 감사합니당 ☺